src/docs/arch-rts.dox

```diff
@@ -0,0 +1,112 @@
+/*! @arch_page arch-rts Rollback to stable
+
+Rollback to stable is an operation that retains only the modifications that
+are stable according to the stable timestamp and recovered checkpoint snapshot.
+
+@section rts-overview Overview of rollback to stable
+
+Rollback to stable process scans each and every table present in the database
```

Comment:
Rollback to stable scans each and every table?

Reply:
Fixed.